### ISSUE TITLE
Tailwind CSS Intellisense VS Code Extension

### ISSUE THREAD

**Issue by slacker-king** [jorenvanhee/tailwindcss-debug-screens#13]
Is this project compatible with the Tailwind CSS Intellisense VS Code Extension?

I am trying out the [Tea Stack](https://github.com/mattwaler/tea-stack) and when I open the project in VS Code and open the package.json file, I get the following error.

![image](https://user-images.githubusercontent.com/23318158/160033396-a77aa275-104f-45d7-81a1-96332a602189.png)

**Comment by slacker-king** [comment]
Never mind, I found out the issue. Apparently the Tea Stack repo is geared for Linux/Mac only.

For example, the .nvmrc won't work with the Windows version of nvm.

And then I ran into the issue with tailwindcss-debug-screens because the npm variables in the package.json file are for Linux/Mac. since the are named **$variable_name** and Windows expects **%variable_name%**.

Sorry for the confusion.
